cli/packages/cmd/user.go

```diff
@@ -0,0 +1,212 @@
+package cmd
+
+import (
+	"errors"
+	"net/url"
+
+	"github.com/Infisical/infisical-merge/packages/config"
+	"github.com/Infisical/infisical-merge/packages/models"
+	"github.com/Infisical/infisical-merge/packages/util"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var userCmd = &cobra.Command{
+	Use:                   "user",
+	Short:                 "Used manage user credentials",
+	DisableFlagsInUseLine: true,
+	Example:               "infisical user",
+	Args:                  cobra.ExactArgs(0),
+	Run:                   func(cmd *cobra.Command, args []string) {},
+}
+
+var switchCmd = &cobra.Command{
+	Use:                   "switch",
+	Short:                 "Used to switch between Infisical profiles",
+	DisableFlagsInUseLine: true,
+	Example:               "infisical switch",
+	Args:                  cobra.ExactArgs(0),
+	PreRun: func(cmd *cobra.Command, args []string) {
+		util.RequireLogin()
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		//get previous logged in profiles
+		loggedInProfiles, err := getLoggedInUsers()
+		if err != nil {
+			util.HandleError(err, "[infisical user switch]: Unable to get logged Profiles")
+		}
+
+		//prompt user
+		profile, err := LoggedInUsersPrompt(loggedInProfiles)
+		if err != nil {
+			util.HandleError(err, "[infisical user switch]: Prompt error")
+		}
+
+		//write to config file
+		configFile, err := util.GetConfigFile()
+		if err != nil {
+			util.HandleError(err, "[infisical user switch]: Unable to get config file")
+		}
+
+		configFile.LoggedInUserEmail = profile
+
+		//set logged in user domain
+		ok := util.ConfigContainsEmail(configFile.LoggedInUsers, profile)
+
+		if !ok {
+			//profile not in loggedInUsers
+			configFile.LoggedInUsers = append(configFile.LoggedInUsers, models.LoggedInUser{
+				Email:  profile,
+				Domain: config.INFISICAL_URL,
+			})
+			//set logged in user domain
+			configFile.LoggedInUserDomain = config.INFISICAL_URL
+
+		} else {
+			//exists, set logged in user domain
+			for _, v := range configFile.LoggedInUsers {
+				if profile == v.Email {
+					configFile.LoggedInUserDomain = v.Domain
+					break
+				}
+			}
+		}
+
+		err = util.WriteConfigFile(&configFile)
+		if err != nil {
+			util.HandleError(err, "")
+		}
+	},
+}
+
+var domainCmd = &cobra.Command{
+	Use:                   "domain",
```

Comment:
what do you think about putting command under `update` subcommand like this `infisical user update domain`?

Reply:
Right, wouldn't be an issue, its more user readable